src/components/SUSHIControls.js

```diff
@@ -264,6 +328,27 @@ export default function SUSHIControls(props) {
             </Button>
           </DialogActions>
         </Dialog>
+        <Dialog
+          open={openExamples}
+          onClose={handleCloseExamples}
+          aria-labelledby="form-dialog-title"
+          maxWidth="lg"
+          fullWidth
+          classes={{ paper: classes.dialogPaper }}
+        >
+          <DialogTitle id="form-dialog-title">Examples</DialogTitle>
+          <DialogContent>
+            <DialogContentText>Use our pre-created examples to learn FSH and get swimming!</DialogContentText>
+            <Grid container direction="row" justify="space-between" alignItems="flex-start" spacing={0}>
+              <React.Fragment>{ExampleGrid()}</React.Fragment>
```

Comment:
I've never seen this `React.Fragment` before. I'll look up some documentation about it! There is also potentially a way you can do this without calling a function for the grid, but we can talk about that and decide if it's something we want to do now.